src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -48,6 +49,16 @@ public class JdbcGtfsExporter {
     // The reference feed ID (namespace) to copy.
     private final String feedIdToExport;
     private List<String> emptyTableList = new ArrayList<>();
+    // These files, even if empty, are required as part of the GTFS specification.
```

Comment:
Would it be possible to pull these from `gtfs.yml`? Or are we not using that anymore

Reply:
I don't think this is used anymore? @binh-dam-ibigroup thoughts?

Reply:
`gtfs.yml` is not in this repo.